### ISSUE TITLE
TypeError: Converting circular structure to JSON (but only with my website)

### ISSUE THREAD

**Issue by alberto56** [pa11y/pa11y#643]
## Expected behaviour

I would expect pa11y to not display "TypeError: Converting circular structure to JSON" with my website, https://www.ahoramoney.org

## Actual behaviour

pa11y works fine with all sites I tried it on (here is an example with amazon) but not with my site, https://www.ahoramoney.org 

    # pa11y --version
    6.2.0
    # pa11y https://amazon.com
    • Error: Duplicate id attribute value "hTKVfD4WL03bf98ZlXOYAQ" found on the web page.
      ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
      ├── #hTKVfD4WL03bf98ZlXOYAQ
      └── <div id="hTKVfD4WL03bf98ZlXOYAQ" class="a-section a-spacing-none shogun-widget deals-shoveler fresh-shoveler"><div class="a-cardui-header as-...</div>

    • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
      ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
      ├── #rhf-container > div:nth-child(1) > table > tbody > tr > td > img
      └── <img src="https://images-na.ssl-images-amazon.com/images/G/01/personalization/ybh/loading-4x-gray._CB485916920_.gif">

    • Error: Anchor element found with a valid href attribute, but no link content has been supplied.
      ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
      ├── #navFooter > div:nth-child(4) > span:nth-child(1) > div > a
      └── <a href="/?ref_=footer_logo"> <div class="nav-logo-b...</a>

    • Error: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
      ├── WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
      ├── #be > form
      └── <form name="ue_backdetect" action="get"><input type="hidden" name="ue_b...</form>

    # pa11y https://www.ahoramoney.org

    Welcome to Pa11y

     > Running Pa11y on URL https://www.ahoramoney.org
    /app/node_modules/pa11y/lib/pa11y.js:164
    		options.log.debug(`Browser Console: args: ${JSON.stringify(message._args)}`);
    		                                                 ^

    TypeError: Converting circular structure to JSON
        --> starting at object with constructor 'BrowserContext'
        |     property '_browser' -> object with constructor 'Browser'
        --- property '_defaultContext' closes the circle
        at JSON.stringify (<anonymous>)
        at state.consoleCallback (/app/node_modules/pa11y/lib/pa11y.js:164:52)
        at /app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:62
        at Array.map (<anonymous>)
        at Object.emit (/app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:43)
        at Page.emit (/app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/puppeteer/common/EventEmitter.js:72:22)
        at Page._addConsoleMessage (/app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:778:14)
        at Page._onConsoleAPI (/app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:721:14)
        at /app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:151:63
        at /app/node_modules/pa11y/node_modules/puppeteer/lib/cjs/vendor/mitt/src/index.js:51:62

## Steps to reproduce

    docker run --rm dcycle/pa11y:1 https://www.ahoramoney.org
    
## Environment

```
docker run --rm dcycle/pa11y:1 --environment

  System:
    OS: Linux 4.15 Debian GNU/Linux 10 (buster) 10 (buster)
    CPU: (2) x64 DO-Regular
    Memory: 362.95 MB / 3.85 GB
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 16.12.0 - /usr/local/bin/node
    Yarn: 1.22.15 - /usr/local/bin/yarn
    npm: 8.1.0 - /usr/local/bin/npm
  pa11y: 6.2.0
```

**Comment by josebolos** [comment]
Hi @alberto56,

Thank you for reporting the bug. This issue should be fixed in pa11y 6.2.2. Could you please try with the latest version and report back?

**Comment by alberto56** [comment]
Thanks for the prompt reply. Version 6.2.2 works fine!
